src/lib/types/minions.ts

```diff
@@ -193,6 +193,8 @@ export interface WoodcuttingActivityTaskOptions extends ActivityTaskOptions {
 	fakeDurationMax: number;
 	fakeDurationMin: number;
 	powerchopping: boolean;
+	forestry: boolean;
+	twitchers: string;
```

Comment:
This should be strictly typed.

Sidenote: Ideally, they would be optional, and the key doesnt exist at all if its false or not in use, as it will save a bit of storage in the db.

For example, instead of storing `{ forestry: false }` in the db, we can just not have that key in the object at all, so `forestry?: true`. For twitchers, it can be the same thing. And the default would go in the task, instead of the default being stored in the data here.

Reply:
I believe I fixed this with the proper solution. I made them both optional and used the typed instead of string for twitchers gloves.